volumes/frus1882.xml

```diff
@@ -36984,7 +36984,7 @@
                                 apaches.</hi></head>
                             <opener><dateline rendition="#right"><placeName><hi rend="smallcaps"
                                             >Department of State and of War and Marine,
-                                        Mexico.</hi></placeName> [<date when="1882-05-04"
+                                        Mexico</hi></placeName>[, <date when="1882-05-04"
```

Comment:
Please confirm the spacing here (if the website can't open the link, it's line 36987 of frus1882).

Reply:
It's correct.  The date portion was added to the existing dateline so I've surrounded the brackets around the added text: [, May 4, 1882]